### PR TITLE
feat(orc8r): Added UT for nil NGC ptr fix and updated example values in swagger

### PR DESCRIPTION
**Pull request by anusha-konda**
Signed-off-by: anusha-konda <anusha.konda@wavelabs.ai>

## Summary

Updated UT for the fix provided for nil NGC ptr in #10717 .
Updated swagger.v1.yml with example values for gateway ngc configs.

## Test Plan

Added new UT to verify the optional NGC Configs added during gateway registration.
Verified lte/{network_id}/gateways POST endpoint, te/{network_id}/gateways/{gateway_id} PUT endpoints with example values of NGC configs.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by anusha-konda** [comment]
Gateway NGC Config example values in Swagger UI
gateway-ngc-example-values
I

File: lte/cloud/go/services/lte/servicers/builder_servicer_test.go [hunk 1]
```diff
@@ -1711,6 +1777,34 @@ func newGatewayConfigNonNat(vlan string, sgi_ip string, sgi_gw string, sgi_ipv6
...
+		DNS: &lte_models.GatewayDNSConfigs{
+			DhcpServerEnabled: swag.Bool(true),
+			EnableCaching:     swag.Bool(false),
+			LocalTTL:          swag.Int32(0),
+		},
+		HeConfig: &lte_models.GatewayHeConfig{},
+	}
+}
+
+func newGatewayNgcConfig() *lte_models.GatewayCellularConfigs {
```
Comment (hcgatewood) [inline lte/cloud/go/services/lte/servicers/builder_servicer_test.go]:
```suggestion
func newGatewayConfigWithNGC() *lte_models.GatewayCellularConfigs {
```

**Review by hcgatewood — APPROVED** [review]
Thanks for the fix!

File: lte/cloud/go/services/lte/servicers/builder_servicer_test.go (hunk 1 above)
Comment (anusha-konda) [reply lte/cloud/go/services/lte/servicers/builder_servicer_test.go]: @hcgatewood , done.

**Pull request merged**